README.md

```diff
@@ -62,10 +62,27 @@ human-readable way with the `printlog` command:
 
     $ bazel-bin/remote_client --grpc_log PATH_TO_LOG printlog
 
+## Getting a list of failed actions
+
+This is available to Remote API V2 functionality, which is present in Bazel 0.17
+and onward.
+
+The remote tool can analyse the grps log and print a list of digests of failed
+actions:
+
+    $ bazel-bin/remote_client --grpc_log PATH_TO_LOG failed_actions
+
+For the purpose of this command, a failed action is any action whose execute
+response returned a non-zero status. This does not include actions whose
+*execution* failed, for example, when remote execution was unavailable or
```

Comment:
But are we sure it's the right thing to do? Maybe we should include such actions as well? After all, from users' POV, these are failures.

Reply:
Oh, I didn't mean gRPC errors, I meant the server errors that we return in the response status field. Some of these can be a result of a user error, for example PERMISSION_DENIED (user forgot to set docker permissions) or our INTERNAL_ERROR in general can indicate a lot of things. But most notably DEADLINE_EXCEEDED! This is something that can definitely behave differently on a bot vs locally and users should be able to debug it. In fact, if you do nothing else, you should have an exception for this one status, I think.

Reply:
You're right. Done.